gdal/ogr/ogrsf_frmts/mdb/ogrmdbjackcess.cpp

```diff
@@ -162,9 +167,41 @@ int OGRMDBJavaEnv::Init()
     {
         JavaVM* vmBuf[1];
         jsize nVMs;
+        int ret = 0;
+
+#if JVM_LIB_DLOPEN
+        const char *jvmLibPtr = "libjvm.so";
+        char jvmLib[PATH_MAX];
+
+        FILE *javaCmd = popen("\"${JAVA_HOME}${JAVA_HOME:+/bin/}java\" -XshowSettings 2>&1 | sed -n '/\\bsun\\.boot\\.library\\.path =/s:.* = \\(.*\\):\\1/server/libjvm.so:p'", "r");
+
+        if (javaCmd != NULL)
+        {
+            size_t javaCmdRead = fread(jvmLib, 1, PATH_MAX, javaCmd);
+            ret = pclose(javaCmd);
+
+            if (ret == 0 && javaCmdRead >= 2)
+            {
+                /* Chomp the new line */
+                jvmLib[javaCmdRead - 1] = '\0';
+                jvmLibPtr = jvmLib;
+            }
+        }
+
+        jint (*pfnJNI_GetCreatedJavaVMs)(JavaVM **, jsize, jsize *);
+        pfnJNI_GetCreatedJavaVMs = (jint (*)(JavaVM **, jsize, jsize *))
+            CPLGetSymbol(jvmLibPtr, "JNI_GetCreatedJavaVMs");
+
+        if (pfnJNI_GetCreatedJavaVMs == NULL)
```

Comment:
Can you add a CPLDebug("MDB", "Cannot find JNI_GetCreatedJavaVMs function" ) message here ?